homematicip/home.py

```diff
@@ -775,6 +775,8 @@ def _ws_on_message(self, message):
                     data = event["home"]
                     obj = self
                     obj.from_json(data)
+                    obj._get_functionalHomes(data)
+                    obj._load_functionalChannels()

```

Comment:
nice catch.
could you please change it to obj.update_home_only(data)
this method will call from_json and the other 2 methods together.

Reply:
Done